### PR TITLE
Cosmetic updates from User Test 1

### PR DESCRIPTION
**Pull request by garrettmflynn**
This PR implements a set of cosmetic updates from User Test 1—specifically the inclusion of subtitles to indicate page purpose.

### CODE REVIEW

File: src/renderer/src/stories/pages/guided-mode/setup/GuidedNewDatasetInfo.js [hunk 1]
```diff
@@ -89,8 +93,14 @@ export class GuidedNewDatasetPage extends Page {
             info.title = `${info.label} Global Metadata`;
             return info;
         });
+        console.log(pages);
 
-        pages.forEach((page) => this.addPage(page.info.label, page));
+        pages.forEach((page) => {
+            page.header = {
+                subtitle: `Uniform ${page.info.label} metadata across all sessions`,
```
Comment (CodyCBakerPhD) [inline src/renderer/src/stories/pages/guided-mode/setup/GuidedNewDatasetInfo.js:100]: I'd suggest making this a bit richer, less of a one-line description and more of a 'how to use this page'

Something like

"Enter any file-level metadata that can serve as the common default for each session."

File: src/renderer/src/stories/pages/guided-mode/setup/GuidedSubjects.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ export class GuidedSubjectsPage extends Page {
     }
 
     header = {
-        subtitle: "Cross-session metadata applied to the final file metadata",
+        subtitle: "Uniform metadata for the subjects included in this conversion",
```
Comment (CodyCBakerPhD) [inline src/renderer/src/stories/pages/guided-mode/setup/GuidedSubjects.js:16]: Likewise to above, would suggest it to be more instructive to the intended purpose

"Enter any subject-level metadata that can serve as the common default across each experiment session."

File: src/renderer/src/stories/pages/guided-mode/setup/GuidedNewDatasetInfo.js [hunk 3]
```diff
@@ -28,6 +28,10 @@ export class GuidedNewDatasetPage extends Page {
 
     #nameNotification;
 
+    header = {
+        subtitle: "Essential metadata for pipeline creation",
```
Comment (CodyCBakerPhD) [inline src/renderer/src/stories/pages/guided-mode/setup/GuidedNewDatasetInfo.js:32]: "Enter a name for this pipeline and specify the base folders to save all outputs to"

I would also enhance the descriptions on this page to specifically explain that stub files are expected to much smaller than the true files (only ~MB in scale); also we might want to rename 'stub' here to 'preview' or 'temporary preview' files for consistency with how we refer to it in the guided mode itself

File: src/renderer/src/stories/pages/guided-mode/data/GuidedStructure.js [hunk 4]
```diff
@@ -29,6 +29,10 @@ export class GuidedStructurePage extends Page {
         this.searchModal.appendChild(this.search);
     }
 
+    header = {
+        subtitle: "Required interfaces for this conversion",
```
Comment (CodyCBakerPhD) [inline src/renderer/src/stories/pages/guided-mode/data/GuidedStructure.js:33]: This is not entirely true - someone ought to be able to select an interface here that might not have data available for some sessions

I've not tested if this is possible in the pipeline but we should at least indicate the intended behavior in the description

Suggestion: "Select all interfaces which apply to this experiment"

File: src/renderer/src/stories/pages/guided-mode/setup/GuidedNewDatasetInfo.js (hunk 1 above)
Comment (CodyCBakerPhD) [reply src/renderer/src/stories/pages/guided-mode/setup/GuidedNewDatasetInfo.js:100]: Though I see this is also applies to more than one page; I guess "Enter any {page.info.label}-level metadata that can serve as the common default across each experiment sessions"

File: src/renderer/src/stories/pages/guided-mode/data/GuidedSourceData.js [hunk 5]
```diff
@@ -19,6 +19,10 @@ export class GuidedSourceDataPage extends ManagedPage {
         merge(this.localState, this.info.globalState);
     };
 
+    header = {
+        subtitle: "Essential information about the source files for this conversion",
```
Comment (CodyCBakerPhD) [inline src/renderer/src/stories/pages/guided-mode/data/GuidedSourceData.js:23]:
```suggestion
        subtitle: "Specify the file and folder locations on your local system for each interface, as well as any additional details that might be required",
```

**Comment by CodyCBakerPhD** [comment]
Overall I really like this improvement; current suggestions are all related to how to phrase these descriptors for maximum effect

Also, the inspector and preview pages do not seem to have descriptions; we should add some to those too

"The NWB Inspector has scanned your files for adherence to the Best Practices" (if you can add hyperlinks to the text, feel free to link 'Best Practices' to https://nwbinspector.readthedocs.io/en/dev/best_practices/best_practices_index.html)

"Preview file contents on truncated files using the Neurosift application\nNeurosift can also be run directly on the full files once uploaded to DANDI, or on any file available on your local system"

**Comment by CodyCBakerPhD** [comment]
I'd also say the final DANDI upload page (the summary report) is rich enough to not need a description on its own

**Comment by garrettmflynn** [comment]
@CodyCBakerPhD Alright updated based on your suggestions!

**Comment by CodyCBakerPhD** [comment]
Looks great!

**Review by CodyCBakerPhD — APPROVED** [review]

**Pull request merged**